src/utils/common.js

```diff
@@ -131,6 +132,15 @@ export function getCurrencySymbol(currency: string): string {
   return currencies[currency] || '';
 }
 
+export function formatFiat(src: number | string, baseFiatCurrency?: ?string): string {
```

Comment:
It would be good to create some simple tests for this function

Reply:
okay, will add some :)